aut_docs/Installation_Guide_WindowsOS.md

```diff
@@ -0,0 +1,230 @@
+## Installation Guide for Windows OS
+
+### A one-time setup:
+
+- Download [Python 3.6.5](https://www.python.org/downloads/release/python-365) (Windows x86-64 executable installer version 3.6.5) 
```

Comment:
Why not the latest python version?

Reply:
@vj-codes actually I went through this [yaml](https://github.com/anitab-org/vms/blob/develop/.travis.yml) file it says python 3.6.5 that's why I used this also I guess will it be compatible with all the files in requirements.txt

Reply:
Ahh I don't think so , let's keep it to 3.6.5 only :)